bookkeeper-server/src/main/java/org/apache/bookkeeper/replication/Auditor.java

```diff
@@ -161,6 +174,12 @@
         help = "the number of delayed-bookie-audits cancelled"
     )
     private final Counter numDelayedBookieAuditsCancelled;
+    @StatsDoc(
+        name = METADATA_CHECK_ENSEMBLE_NOT_ADHERING_TO_PLACEMENT_POLICY_COUNTER,
+        help = "total number of "
+            + "segments of ledgers failed to adhere to EnsemblePlacementPolicy found in metadata check"
+    )
```

Comment:
What is the advantage we are getting counting each segment rather than one count per ledger? Just thinking aloud here. If we count by segment, we have no way of knowing what percentage of the store is wack. @reddycharan  do you see any advantage of counting segments instead of just one increment per ledger?

Reply:
by definition ensemble of bookies is associated to segment, so I opted for per segment. Being said that I see point in your concern. Probably its best to log for each segment failure and have metric per ledger.